README.rst

```diff
@@ -38,7 +38,23 @@ below, Bounter uses 31x less memory compared to ``Counter``.
 Bounter is also marginally faster than the built-in ``dict`` and
 ``Counter``, so wherever you can represent your **items as strings**
 (both byte-strings and unicode are fine, and Bounter works in both
-Python2 and Python3), there's no reason not to use Bounter instead.
+Python2 and Python3), there's no reason not to use Bounter instead except:
+
+When not to use Bounter?
+------------------------
+
+Beware, Bounter is only a probabilistic frequency counter and cannot be relied on for fine counting. (You can't expect a data structure with finite size to hold infinite data.)
```

Comment:
Is this file auto-generated? Or why are we keeping two parallel READMEs?

Anyway, the same changes here please.

Reply:
In this case, I propose to stay only `.rst` (because this used for PyPI too) and drop markdown version (to don't support 2 almost same versions)

Reply:
I wonder if they're out of sync already… @aneesh-joshi can you check?

Reply:
@piskvorky 
I will check. The .rst is not auto generated. I manually edited it.